.github/CONTRIBUTING.md

```diff
@@ -1,25 +1,25 @@
 Contributing
 ------------------
 
-Thank you for considering contributing to [Amido](https://amido.com/) Stacks. It's people like you that continue to make Stacks the best it can be.
+Thank you for considering contributing to [Ensono](https://amido.com/) Stacks. It's people like you that continue to make Stacks the best it can be.
 
 Following these guidelines helps you communicate and participate with the developers maintaining and developing these open source projects. We will cover:
 * How to file issues
 * How to raise a PR
 * How to suggest a new feature
 
 
-### Contributing to Amido Stacks
+### Contributing to Ensono Stacks
 
-We love to receive contributions. There are many ways in which you can experience the warm fuzzies of open source contribution, from improving the documentation, writing tutorials or blog posts, submitting issues and feature requests, or writing code which can be incorporated into Amido Stacks.
+We love to receive contributions. There are many ways in which you can experience the warm fuzzies of open source contribution, from improving the documentation, writing tutorials or blog posts, submitting issues and feature requests, or writing code which can be incorporated into Ensono Stacks.
 
 However, we have a few baselines worth noting:
 * Please don't expect your new feature to be implemented unless you or someone else sends a pull request. Submit with a test to get it merged quicker.
 * If you have an issue, please tell us how to reproduce it. Bonus points for a failing test case.
 * Not everything will be fixed.
 * Did we mention we love tests?
 
-Please, don't use the issue tracker if your problem is not strictly Amido Stacks specific. Stack Overflow or other sources are worth considering.
+Please, don't use the issue tracker if your problem is not strictly Ensono Stacks specific. Stack Overflow or other sources are worth considering.
 
 _See the individual [Amido projects](./README.md) for in depth information about environment setup, testing, and code styles._
```

Comment:
Does this one need changing? `[Amido projects]`

Reply:
Oh good spot that man

Reply:
updated